geopandas/io/tests/test_file.py

```diff
@@ -1,24 +1,23 @@
-from collections import OrderedDict
 import datetime
-from distutils.version import LooseVersion
 import os
 import pathlib
 import sys
+from collections import OrderedDict
+from distutils.version import LooseVersion
 
-import fiona
 import numpy as np
+
+import fiona
+import geopandas
 import pandas as pd
+import pytest
 from shapely.geometry import Point, Polygon, box
 
-import geopandas
```

Comment:
Just a question, do you have some automatic (isort?) hook set up that changes all those imports? If that is the case, maybe better turn it off in the geopandas code base as it gives quite some unrelated changes.

Reply:
Yeah, I do. I have it disabled now.

Reply:
BTW, while I just asked to turn off automatic formatting .. ;) 
I would personally recommend using git pre-commit hook when working on geopandas (we have configuration for it included in the repo). That way you won't need to add commits with "blacken", as that gets done automatically. And it ensures it are the same checks as done online.

Reply:
I normally have all of that configured. But, it is different for each project. I didn't have one specific to geopandas setup and was trying to get this in quick - I figured it would be a small change 🙃. But, probably would have been worth the time to get setup.